roles/opencast-packages-install/tasks/main.yml

```diff
@@ -18,7 +18,14 @@
   when: ansible_distribution == "Debian" or ansible_distribution == "Ubuntu"
 
 - import_tasks: centos.yml
-  when: ansible_distribution == "CentOS" or ansible_distribution == "Red Hat Enterprise Linux"
+  when: ansible_distribution == "CentOS" or ansible_distribution == "Red Hat Enterprise Linux" or ansible_distribution == "Amazon"
+
+
+- name: Enable Opencast to start at bootup
```

Comment:
Isn't this part of the packages by default?

Reply:
You had enabled set before. I've removed it from the handler as it is a persistent state rather than transient like "restarting". Again the vendor preset is disabled:

```
$ systemctl status opencast
● opencast.service - Opencast
   Loaded: loaded (/usr/lib/systemd/system/opencast.service; enabled; vendor preset: disabled)
```

The purpose is to make sure opencast is not started when building the AMI but does start when the instance is created from the AMI. The override.conf file makes sure opencast starts after the cloud init  script has done its thing on the first boot.